### DISCUSSION TITLE
part-ii/2008-95/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#58] [Question comments]
Part II 2008 Probability and Measure: 1.II $. 25 \mathrm{~J} \quad$ 

https://questions.tripos.org/part-ii/2008-95/

**Comment by stevezshen** [comment]
For the last part, I got $m(B) = 0$ except $\mathbb{R}$.

**Reply by stevezshen** [reply]
I did Fatou, then Kronecker. So last part is no because of countable subadditivity.
